src/main.js

```diff
@@ -45,22 +53,26 @@ const renderBoard = () => {
 
   const tasksListElement = boardContainerElement.querySelector(`.board__tasks`);
 
-  const makeShowNewTasksFunc = () => {
-    let tasksArr = tasks.slice();
-    return (count) => {
-      tasksArr.splice(0, count).forEach((task) => renderTask(tasksListElement, task));
-      if (!tasksArr.length) {
-        loadMoreBtnElement.remove();
-      }
+  if (!tasks.length) {
+    render(tasksListElement, new NoTasksComponent().getElement());
+  } else {
```

Comment:
Раз дело происходит в конце функции, можно сделать так:

```suggestion
  if (!tasks.length) {
    render(tasksListElement, new NoTasksComponent().getElement());
    return;
  }
```

Тогда `else` не будет нужен, а общий уровень вложенности уменьшится.